### PR TITLE
ci: Enable Linux ARM64 on build and release

### PR DESCRIPTION
**Pull request by marysaka**
This enable building and releases for Linux ARM64 as the core native dependencies are now built for it.

### CODE REVIEW

**Comment by gdkchan** [comment]
Looks like the Linux builds are gone?

**Comment by marysaka** [comment]
> Looks like the Linux builds are gone?

Should be good now

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -10,28 +10,17 @@ env:
 
 jobs:
   build:
-    name: ${{ matrix.OS_NAME }} (${{ matrix.configuration }})
-    runs-on: ${{ matrix.os }}
+    name:  ${{ matrix.platform.name }} (${{ matrix.configuration }})
```
Comment (gdkchan) [inline .github/workflows/build.yml:13]:
```suggestion
    name: ${{ matrix.platform.name }} (${{ matrix.configuration }})
```

**Review by gdkchan — APPROVED** [review]
lgtm, minus comment.

**Pull request merged**